Assets/Scripts/Flow/NoonIsland.cs

```diff
@@ -82,12 +82,10 @@ private void InitUI()
     private void InitPlayer()
     {
         Camera _ = Instantiate(mainCamera);
-        CinemachineVirtualCamera virtualCam = Instantiate(virtualCamera);
+        CameraManager camManager = Instantiate(cameraManager);
```

Comment:
To keep this consistent with the code below. I use this for save stuff. It's not too relevant here, but good practice to be consistent.
```suggestion
        CameraManager camManager = Instantiate(cameraManager);
        Game.AddManager(camManager);
```